src/atomizer.js

```diff
@@ -175,7 +175,7 @@ module.exports = {
      * @return {object} An atomizer config object
      */
     mergeConfigs: function (configs) {
-        return _.merge.apply(null, configs.concat(utils.handleMergeArrays));
+        return Array.isArray(configs) && configs.length > 0 ? _.merge.apply(null, configs.concat(utils.handleMergeArrays)) : {};
```

Comment:
I think we should. We should use http://polyester-site.rc.staging.manhattan.corp.bf1.yahoo.com/